css/pages/about.scss

```diff
@@ -1,2 +1,46 @@
 // About
+.page--about {
+	margin-left: 50px;
+	margin-right: 50px;
 
+	.section__title {
+		text-align: center;
+	}
+
+	.table {
+		width: 100%;
+		height: 320px;
+
+		img {
+			width: 30%;
+			float: left;
+			margin-right: 1%;
+		}
+
+		.desc {
+			text-align: left;
+			width: 100%;
+			height: 280px;
+			margin-bottom: 1%;
+
+
+			h4 {
+				font-size: 18px;
+	            font-weight: 600;
+	            color: teal;
+			}
+			
+			@media (max-width:$s__breakpoint--large) {
+    		text-align:center;
```

Comment:
Can you fix the indentation in this file and also make sure everything is 2 space indents?